### DISCUSSION TITLE
Deployment Queued for 2 hours

### DISCUSSION THREAD

**Discussion by notjaneq** [vercel/community#7047] [Help]
### Summary

Hello, my deployment is queued for 2 HOURS!!!!
There isnt any other active deployment please help me im losing my mind

### Steps to Reproduce

Hello, my deployment is queued for 2 HOURS!!!!
There isnt any other active deployment please help me im losing my mind

**Comment by aimahusnain** [comment]
Make another git repo and deploy it again it will fix your problem.
